packages/gatsby-theme-docs/package.json

```diff
@@ -29,6 +29,7 @@
     "@availity/api-axios": "^5.4.4",
     "@availity/api-core": "^6.0.1",
     "@availity/localstorage-core": "^3.0.0",
+    "@availity/page-header": "^9.0.5",
```

Comment:
Make sure you add this to `devDependencies` for when people are running this project locally